src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -179,10 +179,22 @@ static u8 D_80B65078[] = {
 
 // used in limbdraw
 static Vec3f D_80B65084[] = {
-    { 2000.0f, 4000.0f, 0.0f,},
-    {-1000.0f, 1500.0f, -2000.0f,},
-    {-1000.0f, 1500.0f, 2000.0f,},
-}; 
+    {
```

Comment:
You have some stuff in here that's the same as #315, as evidenced by here and a few other places. You should probably just revert these in your PR so there isn't any confusion.